components/cache/adapters/redis_adapter.rst

```diff
@@ -55,23 +55,25 @@ helper method allows creating and configuring the Redis client class instance us
         'redis://localhost'
     );
 
-The DSN can specify either an IP/host (and an optional port) or a socket path, as well as a
-password and a database index. To enable TLS for connections, the scheme ``redis`` must be
-replaced by ``rediss`` (the second ``s`` means "secure").
+The DSN can specify either an IP/host (and an optional port) or a socket path, as
+well as a database index. To enable TLS for connections, the scheme ``redis`` must
+be replaced by ``rediss`` (the second ``s`` means "secure").
 
 .. note::
 
     A `Data Source Name (DSN)`_ for this adapter must use either one of the following formats.
 
     .. code-block:: text
 
-        redis[s]://[pass@][ip|host|socket[:port]][/db-index]
+        redis[s]://[ip|host|socket[:port]][/db-index]
```

Comment:
if I read the changes from symfony/symfony#45313 correctly, passing a password was possible before. Then reverting #17234 seems to be the solution to me.

Reply:
OK. But this revertt must be applied only to 5.4, right? 6.4 and up should be kept unchanged.

Reply:
yes, that's how I see it as the support for this form was introduced in Symfony 6.1